### PR TITLE
Fix asset diagnostics IDs

### PR DESCRIPTION
**Pull request by IgorMaliukh**
### Description

Fix asset diagnostics IDs.

### Changes made

Fixed Asset diagnostics IDs.

### Checklist

Before review:

- [x] Changelog entry added.
    - Explains the change in `Changed`, `Fixed`, `Added` sections.
- [x] Tests added/changed, if applicable.
    - Functional tests `Area_CanDoX`, `Area_CanDoX_EvenIfYIsTheCase`, `Area_WhenIDoX_AndYHappens_ThisIsTheResult`.
- [x] Docs for new/changed API's.
    - Code is annotated using Xmldoc syntax.

### CODE REVIEW

File: Documentation~/Diagnostics.md [hunk 1]
```diff
@@ -11,8 +11,9 @@ Note that there are different ranges within both Code and Settings diagnostic ID
 - Settings:
   - `PAS0xxx`: Unity settings 
   - `PAS1xxx`: other settings IDs defined in code rather than in the json
-  - `PAT0xxx`: texture related settings
-  - `PAM0xxx`: mesh related settings
+  - `PAA0xxx`: texture related settings
```
Comment (unitystevem) [inline Documentation~/Diagnostics.md:14]: Probably want a new heading in here, saying "Assets", to differentiate between the PAS and PAA issues

**Review by unitystevem — APPROVED** [review]
Would be nice to get the extra heading in Diagnostics.md, but apart from that, this one looks good to me.

File: Documentation~/Diagnostics.md (hunk 1 above)
Comment (IgorMaliukh) [reply Documentation~/Diagnostics.md:15]: Done

**Review by unitystevem — APPROVED** [review]
Looks good

**Pull request merged**